### PR TITLE
More easy and Good to see(The code)

### PR DESCRIPTION
**Pull request by TG-KRISH**
I made some editing here
I think it's nice

### CODE REVIEW

**Review by TG-KRISH — COMMENTED** [review]
It will be nice

File: README.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ pip install -r requirements.txt
...
 
 ## Lichess OAuth
-- Create an account for your bot on [Lichess.org](https://lichess.org/signup)
+- Create an account for your bot on [Lichess](https://lichess.org/signup)
 - NOTE: If you have previously played games on an existing account, you will not be able to use it as a bot account
-- Once your account has been created and you are logged in, [create a personal OAuth2 token](https://lichess.org/account/oauth/token/create) with the "Play as a bot" selected and add a description
+- Once your account has been created and you are logged in, [New Oauth API token](https://lichess.org/account/oauth/token/create) with the "Play as a bot" selected and add a description
 - A `token` e.g. `Xb0ddNrLabc0lGK2` will be displayed. Store this in `config.yml` as the `token` field
-- NOTE: You won't see this token again on Lichess.
+- NOTE: You won't see this token again on Lichess, so try coping it!
```
Comment (ShailChoksi) [inline README.md:50]: `coping` -> `copying`

File: README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ python3 -m pip install -r requirements.txt
 - Edit the variants: `supported_variants` and time controls: `supported_tc` from the config.yml as necessary
 
 ### Windows:
-- Here is a video on how to install the bot: (https://youtu.be/w-aJFk00POQ). Or you may proceed to the next steps.
+- Here is a video on how to install the bot: [click here](https://youtu.be/w-aJFk00POQ). Or you may proceed to the next steps.
```
Comment (icecream17) [inline README.md:27]:
```suggestion md
- Here is [a video on how to install the bot](https://youtu.be/w-aJFk00POQ). Or you may proceed to the next steps.
```

Noooo "click here" is bad for a11y
https://www.google.com/search?client=firefox-b-1-d&q=click+here
First result: https://www.w3.org/QA/Tips/noClickHere

**Review by icecream17 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (icecream17) [reply README.md:27]: Wait why is this resolved, this isn't fixed

**Comment by icecream17** [comment]
The code hasn't actually changed, so the problems are not resolved yet.

**Comment by TheYoBots** [comment]
Do note that this pull request seems to be copying my pull request here https://github.com/ShailChoksi/lichess-bot/pull/336 Which had been made earlier.

**Comment by TG-KRISH** [comment]
ok

**Pull request closed without merging**